### PR TITLE
[MMF-37] More data about event 

### PR DESCRIPTION
**Pull request by agulowaty**
Mobile apps need to have more data about live events. 
I moved REST-specific controllers to separate package so that it does not collide with namespace introduced by old controllers.

Url returned in mainImageUrl is selected from assets provided by Media API. Firstly master image is being probed, then we're falling back to best (in terms of resolution) asset.

New test-scoped dependency added for specs2 JsonMatchers.

I did small refactoring of `GridService`, but it still needs some refactoring towards usage of new Media API endpoint already introduced in https://github.com/guardian/grid/pull/1794.

cc @rtyley, @JonNorman, @regiskuckaertz

Shape of JSON:

```
{
    "events": [
    {
        "id": "21081001844",
            "url": "https://mem.thegulocal.com/event/guardian-live-how-can-we-get-more-people-cycling-in-london-chris-boardman-andrew-gilligan-and-guests-discuss-21081001844",
            "mainImageUrl": "https://media.guim.co.uk/4c3a44853ff4e41c81d7ba15760657caf8219757/0_0_1920_1152/master/1920.jpg",
            "start": {
                "time": "2016-03-10T19:00:00Z",
                "zone": "Europe/London"
            },
            "end": {
                "time": "2016-03-10T20:30:00Z",
                "zone": "Europe/London"
            },
            "venue": {
                "name": "The Guardian",
                "address": {
                    "city": "London",
                    "postCode": "N1 9GU",
                    "country": "GB"
                }
            },
            "title": "Guardian Live | How can we get more people cycling in London? Chris Boardman, Andrew Gilligan and guests discuss"
    },
   ...
```

### CODE REVIEW

**Comment by JonNorman** [comment]
This looks good to me - though I will really only be consuming the `mainImageUrl` attribute. Looking forward to pulling from this when it is live :dancer:

File: frontend/test/controllers/rest/EventsResponseTest.scala [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      val response = EventsResponse(events = Seq(Event.forRichEvent(defaultLiveEvent)))
+
+      asJsonString(response) must /("events") /# 0 / ("mainImageUrl" -> "https://media.guim.co.uk/b7c830bff5104b9ce9951928238fb09004d50335/0_0_1280_768/master/1280.jpg")
+    }
+  }
+
+  def asJsonString(response: EventsResponse): String = Json.stringify(Json.toJson(response))
+}
+
+trait DateTimeFixed extends AroundEach {
```
Comment (rtyley) [inline frontend/test/controllers/rest/EventsResponseTest.scala]: Could you clarify why is this trait necessary? I can't see anything in the JSON of https://mem.thegulocal.com/events.json which looks like it should depend on the invocation time?

File: project/Dependencies.scala [hunk 2]
```diff
@@ -19,12 +19,13 @@ object Dependencies {
   val scalaTest =  "org.scalatestplus" %% "play" % "1.4.0-M4" % "test"
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.1.1"
   val selenium = "org.seleniumhq.selenium" % "selenium-java" % "2.48.2" % "test"
+  val specs2Extra = "org.specs2" % "specs2-matcher-extra_2.11" % "3.6" % "test"
```
Comment (rtyley) [inline project/Dependencies.scala]: Can you use the [`%%`](http://www.scala-sbt.org/0.13/docs/Library-Dependencies.html#Getting+the+right+Scala+version+with) syntax here, ie: `"org.specs2" %% "specs2-matcher-extra" % "3.6" % "test"`

**Comment by rtyley** [comment]
Apart from the minor comments I've made inline, looks good.

File: project/Dependencies.scala (hunk 2 above)
Comment (agulowaty) [inline project/Dependencies.scala]: fixed

File: frontend/test/controllers/rest/EventsResponseTest.scala (hunk 1 above)
Comment (agulowaty) [inline frontend/test/controllers/rest/EventsResponseTest.scala]: removed

**Comment by rtyley** [comment]
Good stuff! @agulowaty , please be aware that when you merge to master on our repos, you're responsible for ensuring that the deploy goes successfully, and also posting a screenshot of your change working successfully in PROD as a final comment on the PR. :fireworks: 

http://status.membership.guardianapis.com/PROD#

**Comment by agulowaty** [comment]
I deployed master manually - for some reason CI didn't pick up my build automagically.

Result as requested:

events-json

Thanks @rtyley

**Pull request merged**